### DISCUSSION TITLE
[Feature Request]: 希望和TMDB一样增加 FANART 的API和IMAGE地址配置

### DISCUSSION THREAD

**Discussion by imaliang** [jxxghp/MoviePilot#3442] [讨论]
### 当前程序版本

v1.6.1

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望和TMDB一样增加 FANART 的API和IMAGE地址配置

如

FANART_API_DOMAIN：
FANART_IMAGE_DOMAIN：

**Comment by faithleysath** [comment]
Fanart有这环境变量？文档里没看到啊
